presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaRecordSet.java

```diff
@@ -84,11 +85,54 @@
 
         ImmutableList.Builder<Type> typeBuilder = ImmutableList.builder();
 
-        for (DecoderColumnHandle handle : columnHandles) {
+        for (DecoderColumnHandle handle : this.columnHandles) {
```

Comment:
Presto generally does not use this to reference class variables, just use columnHandles

Reply:
Fixed.